apps/shared/account/account.component.html

```diff
@@ -0,0 +1,46 @@
+<img (click)="open()" referrerpolicy=“no-referrer” [src]="account.avatarUrl">
```

Comment:
I guess we don't care a lot about high-a11y here, but should this be just a button instead?

Reply:
Yeah, it definitely should, good call.